scripts/deployment_liquidations_peripheral.py

```diff
@@ -0,0 +1,227 @@
+import json
+
+from brownie import (
+    accounts,
+    CollateralVaultCore,
+    CollateralVaultPeripheral,
+    LendingPoolCore,
+    LendingPoolPeripheral,
+    LiquidationsPeripheral,
+    LiquidationsCore,
+    LiquidityControls,
+    LoansCore,
+    Loans,
+    ERC20
+)
+from pathlib import Path
+
+
+NAME_TO_CONTRACT = {
+    "collateral_vault_core": CollateralVaultCore,
+    "collateral_vault_peripheral": CollateralVaultPeripheral,
+    "lending_pool_core": LendingPoolCore,
+    "lending_pool": LendingPoolPeripheral,
+    "liquidations_peripheral": LiquidationsPeripheral,
+    "liquidations_core": LiquidationsCore,
+    "liquidity_controls": LiquidityControls,
+    "loans": Loans,
+    "loans_core": LoansCore,
+    "token": ERC20
+}
+
+
+def load_contracts(env: str) -> dict:
+    config_file = f"{Path.cwd()}/configs/{env}/contracts.json"
+    with open(config_file, "r") as f:
+        contracts = json.load(f)["tokens"]["WETH"]
+
+    return {key: NAME_TO_CONTRACT[key].at(contracts[key]["contract"]) for key in contracts}
+
+
+def dev():
```

Comment:
the dev(), int() and prod() functions seem to do the same thing. Would it make sense to create a new functions that gets as argument the stage and does what these three functions are doing?

Reply:
They do seem to do the same thing for the most part now but depending on the environment, the configuration values of some of the contracts may differ, as they have in the past and may well again for testing purposes in the future. In addition, `prod()` usually has a couple more transactions that aren't needed in `dev()` and `int()`. We can simplify this but at the cost of having to revert that simplification if configuration values change in the future again.

For example, in this specific contract, we may have grace period and lenders period durations that are different for mainnet.

What do you think? @ametade @cfcfs